primitives/src/lib.rs

```diff
@@ -49,3 +49,5 @@ pub struct NFTDetails {
 
 /// How NFT IDs are encoded.
 pub type NFTId = u32;
+/// How the NFT series id is encoded.
+pub type NFTSeriesId = u64;
```

Comment:
Why `u64` vs `u32`? Small difference anyways

Reply:
I was not sure if u32 is going to be large enough. It should be now u32.